fleet/lib/loki/overlays/antu/values.yaml

```diff
@@ -0,0 +1,47 @@
+loki:
+  storage:
+    s3:
+      endpoint: http://rook-ceph-rgw-o11y.rook-ceph.svc.cluster.local:80
+
+  limit_config:
+    query_timeout: 60s
+
+compactor:
+  replicas: 1
+  persistence:
+    enabled: true
+    storageClassName: rook-ceph-block
+
+ingester:
+  replicas: 1
+  persistence:
+    enabled: true
+    size: 15Gi
+    storageClassName: rook-ceph-block
+
+distributor:
+  replicas: 1
+
+querier:
+  replicas: 1
+
+queryFrontend:
+  replicas: 1
+
+queryScheduler:
+  replicas: 1
+
+indexGateway:
+  replicas: 1
+
+gateway:
+  ingress:
+    hosts:
+      - host: loki.antu.ls.lsst.org
```

Comment:
why is this here? if the default values will do the same?
`loki.${ get .ClusterLabels "management.cattle.io/cluster-display-name" }.${ .ClusterLabels.site }.lsst.org`